recipe/meta.yaml

```diff
@@ -1,38 +1,43 @@
+{% set name = "feedgenerator" %}
 {% set version = "2.1.0" %}
 
 package:
-  name: feedgenerator
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/f/feedgenerator/feedgenerator-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/feedgenerator-{{ version }}.tar.gz
   sha256: f075f23f28fd227f097c36b212161c6cf012e1c6caaf7ff53d5d6bb02cd42b9d
 
 build:
   noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.7
+    - setuptools >=40.6.0
+    - wheel
+    - poetry-core
```

Comment:
I'm curious what poetry-core was added here. I don't see any invocation of poetry, pip is still used above. I guess, same goes for wheel.

Reply:
It is declared in their pyproject.toml.

Reply:
I just checked it locally and they aren't needed, so I opened #15.

Reply:
They are if you are not using the setup.py. Many packages are taking this hybrid approach but our recipe regenerator is using the latest best practices to ensure future compatibility.